include/oneapi/tbb/task_group.h

```diff
@@ -417,7 +419,9 @@ class task_group_context : no_copy {
     friend class task_group_base;
 }; // class task_group_context
 
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 1070
 static_assert(sizeof(task_group_context) == 128, "Wrong size of task_group_context");
```

Comment:
here we might have to re calculate the right side of the equation for 32bit/64bit. Not sure if that issue is `MAC_OS_X` only

Reply:
I may need help here :)

For the time-being, I disabled this assert on Darwin PPC, but hopefully we can re-calculate and enable it back.